packages/data-model/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "interop-data-model",
+  "version": "0.1.0",
+  "description": "> TODO: description",
+  "author": "Ángel Araya <angel.araya@janeirodigital.com>",
+  "homepage": "",
+  "license": "MIT",
+  "main": "dist/index.js",
+  "directories": {
+    "lib": "src",
+    "test": "test"
+  },
+  "scripts": {
+    "build": "tsc -b .",
+    "test": "jest",
+    "clean": "rimraf dist tsconfig.tsbuildinfo"
+  },
+  "dependencies": {
+    "interop-client": "^0.1.0",
+    "n3": "^1.10.0"
+  },
+  "jest": {
+    "globals": {
+      "ts-jest": {
+        "tsconfig": "../../tsconfig.json"
+      }
+    },
+    "transform": {
+      "^.+\\.ts$": "ts-jest"
+    },
+    "testRegex": "/test/.*-test.ts$",
+    "moduleFileExtensions": [ "js", "ts" ],
+    "collectCoverage": false
+  },
+  "devDependencies": {
+    "@types/n3": "^1.8.0"
+  }
+}
```

Comment:
Let's keep jest config out of the package.json file. It can be either `jest.config.js` or `jest.config.ts`